shell/server/stats-server.js

```diff
@@ -93,7 +93,7 @@ computeStats = function (since) {
 
     const counts = ApiTokens.aggregate([
       { $match: {
-          "owner.user.lastUsed": timeConstraint,
+          lastUsed: timeConstraint,
```

Comment:
Seems like we should now add `"owner.user": {$exists: true}` to this query, since it's intended to only count user-owned tokens?